cves/CVE-2018-13304.yml

```diff
@@ -276,8 +286,8 @@ i18n:
 
     Answer should be boolean. Write a note about how you came to the conclusions
     you did.
-  answer:
-  note:
+  answer: false
+  note: This is a concurrency issue.
```

Comment:
Did you mention concurrency above? I think this is the first time I've read this, so maybe include it in the main description.